utils/toggle-negation.ts

```
@@ -117,7 +116,7 @@ function toggleLogicalExpression(
   node: LogicalExpression,
   context: Rule.RuleContext,
 ): string {
-  let content = getNodeContent(node, context).trim()
+  let content = context.sourceCode.getText(node).trim()
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `Node | undefined`.

(typescript/no-unsafe-argument)